docs/Changelog.md

```diff
@@ -11461,6 +11461,81 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain to any tensor type.</dd>
 </dl>
 
+### <a name="Pad-11"></a>**Pad-11**</a>
+
+  Given a tensor containing the data to be padded (`data`), a tensor containing the number of start and end pad values for axis (`pads`), (optionally) a `mode`, and (optionally) `value`, 
+  a padded tensor (`output`) is generated.
+  
+  The three supported `modes` are (similar to corresponding modes supported by `numpy.pad`):
+  
+  1) `constant`(default) - pads with a given constant value as specified by `value` (which defaults to 0)
+  
+  2) `reflect` - pads with the reflection of the vector mirrored on the first and last values of the vector along each axis
+  
+  3) `edge` - pads with the edge values of array
+  
+  Example (`constant` mode):
+    Insert 0 pads to the beginning of the second dimension.
+  
+    data = 
```

Comment:
Add example for other modes.

Reply:
Will add

Reply:
Add example for 'reflect' and 'edge' modes. Resolved.